hoot-core/src/main/cpp/hoot/core/io/GeoNamesReader.cpp

```diff
@@ -41,34 +41,75 @@ namespace hoot
 HOOT_FACTORY_REGISTER(OsmMapReader, GeoNamesReader)
 
 GeoNamesReader::GeoNamesReader() :
-_GEONAMESID(0),
-_LATITUDE(4),
-_LONGITUDE(5),
 _maxSaveMemoryStrings(ConfigOptions().getGeonamesReaderStringCacheSize()),
 _defaultCircularError(ConfigOptions().getCircularErrorDefaultValue()),
 _useDataSourceIds(false)
 {
-  _columns << "geonameid";
-  _columns << "name";
-  _columns << "asciiname";
-  _columns << "alternatenames";
-  _columns << "latitude";
-  _columns << "longitude";
-  _columns << "feature_class";
-  _columns << "feature_code";
-  _columns << "country_code";
-  _columns << "cc2";
-  _columns << "admin1_code";
-  _columns << "admin2_code";
-  _columns << "admin3_code";
-  _columns << "admin4_code";
-  _columns << "population";
-  _columns << "elevation";
-  _columns << "dem";
-  _columns << "timezone";
-  _columns << "modification_date";
-
-  _convertColumns << 1 << 3 << 6 << 7 << 8 << 9 << 10 << 11 << 12 << 13 << 14 << 15 << 18;
+  // Geonames.org column names
+//  _columns << "geonameid";
+//  _columns << "name";
+//  _columns << "asciiname";
+//  _columns << "alternatenames";
+//  _columns << "latitude";
+//  _columns << "longitude";
+//  _columns << "feature_class";
+//  _columns << "feature_code";
+//  _columns << "country_code";
+//  _columns << "cc2";
+//  _columns << "admin1_code";
+//  _columns << "admin2_code";
+//  _columns << "admin3_code";
+//  _columns << "admin4_code";
+//  _columns << "population";
+//  _columns << "elevation";
+//  _columns << "dem";
+//  _columns << "timezone";
+//  _columns << "modification_date";
+
+  // Commenting this out for a while.  I'm not sure what it was supposed to do
```

Comment:
<< is the append operator for `QStringList` Guess you don't need it any more. All this commented out code should be cleaned up.

Reply:
That should have been removed. i moved the `_columns` setup from the constructor to the `open` function since that is where the file type decision is made.